src/hotspot/share/jvmci/jvmciRuntime.cpp

```diff
@@ -1806,14 +1806,14 @@ void JVMCIRuntime::get_field_by_index_impl(InstanceKlass* klass, fieldDescriptor
   constantPoolHandle cpool(thread, klass->constants());
 
   // Get the field's name, signature, and type.
-  Symbol* name  = cpool->name_ref_at(index);
+  Symbol* name  = cpool->name_ref_at(index, Bytecodes::_getfield /*We know it's a field*/);
```

Comment:
You should not hard code the bytecode. The bytecode should be passed in by callers such as this place:

```
ciField* ciBytecodeStream::get_field(bool& will_link) {
  ciField* f = CURRENT_ENV->get_field_by_index(_holder, get_field_index());
  will_link = f->will_link(_method, _bc);
  return f;
}
```

(Same comment for the other Bytecodes::_getfield in this function.